src/Service/RoleService.php

```diff
@@ -37,33 +37,19 @@
     /**
      * @var RoleProviderInterface
      */
-    protected $roleProvider;
+    private $roleProvider;
 
     /**
      * @var string
      */
-    protected $guestRole = '';
+    private $guestRole;
 
-    public function __construct(RoleProviderInterface $roleProvider)
+    public function __construct(RoleProviderInterface $roleProvider, string $guestRole)
     {
         $this->roleProvider = $roleProvider;
-    }
-
-    public function setRoleProvider(RoleProviderInterface $roleProvider): void
-    {
-        $this->roleProvider = $roleProvider;
-    }
-
-    public function setGuestRole(string $guestRole): void
-    {
         $this->guestRole = $guestRole;
     }
 
-    public function getGuestRole(): string
```

Comment:
not sure about this one but why we need this? If we need for some reason maybe we can get from `ZfcRbac\Options\ModuleOptions`

Reply:
agree, remove